### PR TITLE
set name of instance_groups

### PR DESCRIPTION
**Pull request by ahaverbuch**
To better support cloning in UI - default to name of role if not set

### CODE REVIEW

File: lib/elasticity/instance_group.rb [hunk 1]
```diff
@@ -119,6 +125,7 @@ def to_aws_instance_config
         :instance_count => @count,
         :instance_type => @type,
         :instance_role => @role,
+        :name => @name || "#{@role}-#{object_id}"  # default to role-objectid to differentiate duplicates
```
Comment (bcbrown) [inline lib/elasticity/instance_group.rb]: What is `object_id` and how is it in scope? I don't see a reference to it.

Comment (ahaverbuch) [inline lib/elasticity/instance_group.rb]: All ruby objects have an object_id. It is the id of particular instance of the ruby class. And thus different for every instance_group in this case

Comment (Catalectic) [inline lib/elasticity/instance_group.rb]: I assume @role is a string, not a symbol?

Comment (ahaverbuch) [inline lib/elasticity/instance_group.rb]: Yes

**Pull request merged**